sys-apps/keyutils/files/tmpfiles.d/keyutils.conf

```diff
@@ -0,0 +1,3 @@
+L	/etc/request-key.conf	-	-	-	-	../usr/share/keyutils/request-key.conf
+L	/etc/request-key.d	-	-	-	-	../usr/share/keyutils/request-key.d
```

Comment:
Please create this as a directory and add symlinks inside it as needed. This may require adjusting other ebuilds to use the /usr/share location instead of relying on the symlink.